app/templates/thank-you.html

```diff
@@ -2,6 +2,8 @@
 
 {% block page_title %}Submission Successful{% endblock %}
 
+{% block subheader %}{% endblock subheader %}
```

Comment:
What is the expectation for the previous link on the introduction and thank you pages?